src/groups/group_action.jl

```diff
@@ -32,6 +32,17 @@ Get the direction of the action
 """
 direction(::AbstractGroupAction{AD}) where {AD} = AD()
 
+@doc raw"""
```

Comment:
Maybe use a reserved order here p, A, a since it is the adjoint?

Reply:
Do you mean `adjoint_apply_diff_group(A::AbstractGroupAction, p, X, a)`?

Reply:
yes, or would that be too irritating?

Reply:
I think we usually try to place tangent vector right after the point it is tangent at and here `X` is tangent at `a` but with so many variants of differentials even to me it's impossible to use them without carefully reading the documentation. So if the `p, X, a` order is more intuitive for you I can use it :slightly_smiling_face:.

Reply:
Ah maybe your reasoning is better, mine is surely not based on much intuition and your argument sounds quite intuitive. So let‘s keep it as is.